backends/qualcomm/runtime/QnnManager.cpp

```diff
@@ -465,16 +475,158 @@ bool QnnManager::IsNodeSupportedByBackend(
   return true;
 }
 
-Error QnnManager::Compile(
-    std::vector<std::shared_ptr<OpWrapper>>& op_wrappers,
+Error QnnManager::GetContextBinary(
     QnnExecuTorchContextBinary& qnn_executorch_context_binary) {
+  ET_CHECK_OR_RETURN_ERROR(
+      backend_params_ptr_->qnn_context_ptr_->GetContextBinary(
+          qnn_executorch_context_binary) == Error::Ok,
+      Internal,
+      "Fail to get context binary.");
+
+  return Error::Ok;
+}
+
+Error QnnManager::CompileQcir() {
+  // check if context binary came from flatbuffer
```

Comment:
If I understand correctly, it's for compiling flatbuffers graph so we can merge the graph AOT?

Reply:
This is used for compiling `qcir`, but the merging of `qcir` happened earlier in `PyQnnManagerAdapter.h`'s constructor which consumes multiple flatbuffers.